### ISSUE TITLE
Windows Defender detected Win32/Wacatac.B!ml in manuskript.exe

### ISSUE THREAD

**Issue by qantravon** [olivierkes/manuskript#877]
I am fairly certain this is a false positive, but is there any way to verify that?

**Comment by TheJackiMonster** [comment]
Yes, this should be a false positive. There's some information about that in a previous issue #112 which is also referenced on the [releases page](https://github.com/olivierkes/manuskript/releases/). However if you want to be sure there is no malware attached to the binary, I recommend [running Manuskript from source code](https://github.com/olivierkes/manuskript/wiki/Run-Manuskript-from-Source-Code-on-Windows). The wiki should explain this pretty straight forward and because Manuskript is all written in Python which shouldn't be too complicated to be run on Windows.

Otherwise you can also try building a binary yourself with one of these options:
 - There is a [guide](https://github.com/olivierkes/manuskript/wiki/Package-Manuskript-for-Windows-with-PyInstaller) in the wiki to do that using Windows.
 - You can try to use the [script](https://github.com/olivierkes/manuskript/blob/develop/package/build_for_windows.sh) I have written to build it from Linux.

Maybe it doesn't end up as false positive if you build everything on Windows but I don't have any to do so. I was also not sure about building the binary for the official release on a Windows partition from anyone I know because I couldn't be sure about any malware on that system. So I thought building the Windows binary on a Linux system without any Windows malware close should be fine. ^^'
